core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -147,35 +218,59 @@ var LearnerDashboardPage = function() {
     expect(await feedbackMessage.first().getText()).toMatch(message);
   };
 
-  this.checkIncompleteExplorationSection = async function(explorationTitle) {
-    await this.navigateToInCompleteSection();
-    await this.navigateToIncompleteExplorationsSection();
-    await this.expectTitleOfExplorationSummaryTileToMatch(
-      explorationTitle);
-  };
+  this.navigateToCommunityLessonsAndCheckIncompleteExplorations = (
+    async function(explorationTitle) {
+      await this.navigateToCommunityLessonsSection();
+      // eslint-disable-next-line oppia/check-element-selector-at-top
+      var explorationTitleInIncompleteSection = element(
+        by.css(
+          '.protractor-test-incomplete-community-lessons-section')).element(
+        by.cssContainingText(
+          '.protractor-test-exp-summary-tile-title', explorationTitle));
+      expect(
+        await explorationTitleInIncompleteSection.getText()).toMatch(
```

Comment:
Can you use `action.getText()` here? Before interacting with any page element, we need to check that it has loaded, which the `action` functions handle for you

ditto elsewhere

Reply:
Done.